pages/_app.js

```diff
@@ -1,7 +1,16 @@
-import '../styles/globals.css'
+import Head from 'next/head';
+import '../styles/globals.css';
 
 function MyApp({ Component, pageProps }) {
-  return <Component {...pageProps} />
+  return (
+    <div>
+      <Head>
+        <title>instagram clone</title>
```

Comment:
Since this is title, should use title [case](https://en.wikipedia.org/wiki/Title_case).
```suggestion
        <title>Instagram Clone</title>
```

Reply:
Done